src/Esfa.Vacancy.Register.UnitTests/GetApprenticeshipVacancy/Api/Mappings/GivenAnApprenticeshipMapper/WhenGettingLiveVacancyWithApprenticeshipMinimumWageType.cs

```diff
@@ -65,7 +65,7 @@ public void ShouldHaveUnknownWageForVacancyWithUndefinedHoursPerWeek()
             var vacancy = _sut.MapToApprenticeshipVacancy(apprenticeshipVacancy);
 
             vacancy.VacancyReference.Should().Be(VacancyReference);
-            vacancy.WageUnit.Should().BeNull();
+            vacancy.WageUnit.Should().Be(WageUnit.Unspecified);
```

Comment:
Whilst you are making changes here, can you also reorganise the test files and names per new conventions?